DataFormats/BTauReco/src/classes_def.xml

```diff
@@ -410,18 +414,24 @@
     <version ClassVersion="3" checksum="386672900"/>
     <version ClassVersion="4" checksum="113477759"/>
   </class>
-  <class name="btagbtvdeep::SecondaryVertexFeatures" ClassVersion="4">
+  <class name="btagbtvdeep::SecondaryVertexFeatures" ClassVersion="6">
+   <version ClassVersion="6" checksum="1869636758"/>
+   <version ClassVersion="5" checksum="1572623975"/>
```

Comment:
This intermediate "5" version is not needed -> move "6" to "5"

Reply:
Thanks for the comment, I was not sure about this when I applied the ClassVersions fix (as in, if one has to increment and keep track of all intermediate versions). I now went ahead with your suggestion to use the latest checksum and denote this as the new version "5".

With that fix applied, building works locally, as if there had never been an intermediate version.